spec/schemas/_common.yaml

```diff
@@ -1796,13 +1796,13 @@ components:
           description: The average slice count of all search requests. This is computed as the total slice count divided by the total number of concurrent search requests.
           type: number
         fetch_current:
-          description: The number of shard fetch operations that are currently running.
+          description: The number of shard fetch operations that are currently running. 
           type: number
         fetch_time:
           description: The total amount of time for all shard fetch operations.
           $ref: '#/components/schemas/Duration'
         fetch_time_in_millis:
-          description: The total amount of time for all shard fetch operations, in milliseconds.
+          description: The total amount of time for all shard fetch operations, in milliseconds. 
```

Comment:
"The total amount of time taken to complete all shard fetch operations, in milliseconds"?